Source/Scene/BingMapsImageryProvider.js

```diff
@@ -533,7 +553,29 @@ define([
         }
         //>>includeEnd('debug');
 
-        return ImageryProvider.loadImage(this, buildImageResource(this, x, y, level, request));
+        var promise = ImageryProvider.loadImage(this, buildImageResource(this, x, y, level, request));
+
+        if(defined(promise)) {
+            return promise.otherwise( (error) => {
+
+                // One possible cause of an error here is that the image we tried to load was empty. This isn't actually
+                // a problem. In some imagery sets (eg. `BingMapsStyle.AERIAL_WITH_LABELS_ON_DEMAND`), an empty image is
+                // returned rather than a blank "This Image is Missing" placeholder image. In this case, we supress the
+                // error.
+                if (defined(error.blob) && error.blob.size === 0) {
+
+                    // If our tile discard policy tells us how to represent an empty image, we return that.
+                    if (defined(this._tileDiscardPolicy.emptyImage)) {
```

Comment:
This is cool but probably unnecessary complexity. Just returning `DiscardEmptyTilePolicy.DEFAULT_EMPTY_IMAGE` is fine.

Reply:
And at that point just rename it to `DiscardEmptyTilePolicy.EMPTY_IMAGE`.

Reply:
What if someone wanted to reuse the DiscardEmptyTilePolicy for another imagery provider though, where the behaviour for an empty tile was different? It seems silly to make them either implement a brand new discard policy or update the code for the existing one, and then you'd have to update this existing code to respect the setting.